### PR TITLE
Giant Form buff

### PR DESCRIPTION
**Pull request by TheUnholyWrath**
4 seconds level 4
2 seconds level 5
probably

attack rate down to 0.7

### CODE REVIEW

File: game/scripts/npc/items/custom/item_giant_form.txt [hunk 1]
```diff
@@ -48,7 +48,7 @@
     "AbilityTextureName"                                  "custom/giant_form_1"
     "AbilityCastPoint"                                    "0.0"
     "AbilityManaCost"                                     "50"
-    "AbilityCooldown"                                     "6.0"
+    "AbilityCooldown"                                     "4.0"
```
Comment (POCKETMONEY99) [inline game/scripts/npc/items/custom/item_giant_form.txt]: Don't you have to write it like "4.0 2.0" Im not 100% sure though. Ask a real dev.

**Comment by POCKETMONEY99** [comment]
This item needs a rework or removing imo. I currently don't see a middle ground where this item is balanaced in its current state.

**Comment by TheUnholyWrath** [comment]
i think you're right i'll fix that, its scabos idea, thats what he wanted changed about it. if its still bad then no big deal life goes on, if its too good can always reel it back in a bit

**Comment by lovesmuffins** [comment]
#LETMELEEMODIFIERSWORKONGIANTFORM
#IWANTMYECHOSABREGIANTFORMBUILDTOBEGOOD

**Comment by SquawkyArctangent** [comment]
echo saber literally can't work with giant form on a fundamental level, because echo saber just gives you a lot of attack speed and giant form sets your attack speed to base

**Comment by lovesmuffins** [comment]
> echo saber literally can't work with giant form on a fundamental level, because echo saber just gives you a lot of attack speed and giant form sets your attack speed to base
 

ye which sucks lol.
It should set your BAT to like 1.5 reduce your attack speed by 75% or something like that. (with a minimum attack speed of 150

**Pull request merged**